omxd.c

```diff
@@ -4,12 +4,14 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
Could you please do it without getopt? It has blown up the size of main, unnecessarily. And introduced a dependency, which you did not handle in the Makefile deb rule for Debian package generation. If you insist on getopt, I insist that you fully update and test the deb generation and install.

I'm a bit allergic to dependencies. omxd at the moment has none, apart from the C-library. And I'd like to keep it that way.